src/main/java/sirius/biz/tenants/mongo/MongoUserAccountImportJobFactory.java

```diff
@@ -22,7 +22,6 @@
  * Provides an import job for {@link MongoUserAccount user accounts} stored in MongoDB.
  */
 @Register(framework = MongoTenants.FRAMEWORK_TENANTS_MONGO)
-@Permission(UserAccountController.PERMISSION_MANAGE_USER_ACCOUNTS)
```

Comment:
Wieso sind die unnötig?

Reply:
checkPermission is overwritten using the logic of UserAccountController below...